config/src/main/java/org/springframework/security/config/annotation/web/configurers/oauth2/server/resource/OAuth2ResourceServerConfigurer.java

```diff
@@ -209,13 +231,57 @@ public void configure(H http) throws Exception {
 			return OAuth2ResourceServerConfigurer.this;
 		}
 
+		public JwtConfigurer timestampTokenValidator(JwtTimestampTokenValidator validator) {
+			this.timestampTokenValidator = validator;
+			return this;
+		}
+
+		public JwtConfigurer addTokenValidators(OAuth2TokenValidator<Jwt, OAuth2Error>... validators) {
+			this.validators = Arrays.asList(validators);
+			return this;
+		}
+
+		public OAuth2ResourceServerConfigurer<H> and() {
+			return OAuth2ResourceServerConfigurer.this;
+		}
+
 		JwtDecoder getJwtDecoder() {
 			if ( this.decoder == null ) {
 				return this.context.getBean(JwtDecoder.class);
 			}
 
 			return this.decoder;
 		}
+
+		JwtTimestampTokenValidator getJwtTimestampTokenValidator() {
```

Comment:
This method is package-private so that it can be unit tested against edge cases.